index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <h1>Hello microverse</h1>
```

Comment:
♻️ Awesome! You are using the required `h1` tag for setting the main title of your page. However, there is a small change you need to do. In order to add the styles from the CSS file you wrote, you need to import that file into your HTML so you can use the `.heading` class on the `h1` tag. In order to do that, you can make use of a [`link` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link). 🤓

- [ ] Kindly, link your CSS file and add the class to your heading. 

Currently, the styling is not being applied:
![image](https://user-images.githubusercontent.com/26911718/204536940-beb8ba07-5e4a-41d5-b6bb-50437ea471c3.png)